pixart.js

```diff
@@ -1 +1,51 @@
+function preventDef(event) {
+  event.preventDefault();
+}
+var color = "1B4370";
+var setColor = document.getElementById("set-color");
+var brush = document.querySelector(".brush");
+var input = document.getElementById("color-field");
+
+function colorChange() {
+	var newColor = input.value;
+	brush.style.backgroundColor = newColor;
+}
+
+//commit 2: This already works with both clicking the submit button and pressing the enter key. Was I supposed to do something else?
+setColor.addEventListener("click", preventDef);
+setColor.addEventListener("click", colorChange);
+
+function makeDivs(){
+  for(var x = 0; x < 20; x++){
+    var newSquare = document.createElement("DIV");
+    newSquare.className = "square";
+    newSquare.id = x;
+    document.querySelector("body").appendChild(newSquare);
+  }
+}
+
+//commit 3: makin' 20 divs
+makeDivs();
+
+var squares = document.querySelectorAll(".square");
```

Comment:
It looks like you were adding in an extra step here. If you want to loop through all the squares and do something to each of them, you could do it like this:
``
var squares = document.querySelectorAll(".square");
for(var x = 0; x < squares.length; x++){
  var square = squares[x];
  square.style.backgroundColor = "antifreeze"; // or whatever you want to do
}
```